src/media/local-stream.ts

```diff
@@ -283,6 +283,32 @@ abstract class _LocalStream extends Stream {
     return this.effects;
   }
 
+  /**
+   * Method to serialize data about input, output streams
+   * and also effects from _LocalStream.
+   *
+   * @returns - A JSON-compatible object representation with data from _LocalStream.
+   */
+  toJSON() {
+    return {
```

Comment:
Maybe another nitpick, but this is returning an objec, not "json", right?  I think if this method is going to be called `toJson` we should `JSON.stringify` and return a `String`.

Reply:
@bbaldino yes and with your idea, it requires you to manually call your custom method each time you need it, and it won't be used automatically by `JSON.stringify()`. That's more transparent. But I also want to ask @marcin-bazyl will it be convenient to use it in this way?

Reply:
@bbaldino  the whole point of this jira is to make JSON.stringify() work, not to invent new methods. Any library/component that uses webrtc-core should be free to call JSON.stringify() on instances of webrtc-core classes like LocalStream and that should not cause the browser to hang up (that's what's currently happening).

Reply:
👍